pkg/pointerdb/pointerdb.go

```diff
@@ -61,7 +61,11 @@ func (s *Server) validateAuth(ctx context.Context) error {
 }
 
 func (s *Server) appendSignature(ctx context.Context) error {
-	signature, err := auth.GenerateSignature(s.identity)
+	if s.identity == nil {
```

Comment:
Under what circumstances would `s.identity == nil`?

I'm not familiar with the expected behavior here but based on the name of the function I would assume you would return an error if there is no identity with which to generate a signature.

Reply:
I wanted to avoid changing tests because with next PR (for https://storjlabs.atlassian.net/browse/V3-591) I will need to reorganize this method (`appendSignature`) one way or another but I think I figure out better way to handle this.